doc/classes/Transform3D.xml

```diff
@@ -181,8 +181,25 @@
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
-			A transform with no translation, no rotation, and its scale being [code]1[/code]. Its [member basis] is equal to [constant Basis.IDENTITY].
-			When multiplied by another [Variant] such as [AABB] or another [Transform3D], no transformation occurs.
+			The identity [Transform3D]. This is a transform with no translation, no rotation, and not scaled. Its [member basis] is equal to [constant Basis.IDENTITY]. This also means that:
```

Comment:
Reason why _"and its scale being [code]1[/code]"_ was changed is because, nitpicky, it's technically incorrect. The "scale" is Vector3.

Reply:
Same here for scale